templates/partials/answers/textfield.html

```diff
@@ -17,4 +17,21 @@
   "legendClasses": "u-vh",
   "mutuallyExclusive": mutuallyExclusive,
   "error": error
-}) }}
+} %}
+
+{% if answer.url %}
+  {% set config = config | setAttribute("typeahead", {
+    "instructions": "Use up and down keys to navigate suggestions once you\'ve typed more than two characters. Use the enter key to select a suggestion. Touch device users, explore by touch or with swipe gestures.",
+    "ariaYouHaveSelected": "You have selected",
+    "ariaMinChars": "Enter 3 or more characters for suggestions.",
+    "ariaOneResult": "There is one suggestion available.",
+    "ariaNResults": "There are {n} suggestions available.",
+    "ariaLimitedResults": "Results have been limited to 10 suggestions. Type more characters to improve your search.",
+    "moreResults": "Continue entering to improve suggestions",
+    "resultsTitle": "Suggestions",
+    "noResults": "No results found",
```

Comment:
Do we need to mark these for translations?. I'm not sure about arias.

Reply:
Yes, I would imagine so.